apps/gitness/src/components/FileEditor.tsx

```diff
@@ -153,158 +153,156 @@ export const FileEditor: React.FC = () => {
+                    <BreadcrumbItem>
+                      <BreadcrumbLink asChild>
+                        <Link to={repoPath + '/~/' + path.parentPath}>
+                          <Text
+                            size={2}
+                            color="tertiaryBackground"
+                            className={cn('hover:text-foreground', {
+                              'text-primary': index === pathParts?.length - 1
+                            })}
+                          >
+                            {path.path}
+                          </Text>
+                        </Link>
+                      </BreadcrumbLink>
+                    </BreadcrumbItem>
+                    <Text size={2} color="tertiaryBackground" className="pt-2">
+                      /
+                    </Text>
+                  </>
+                )
+              })}
+              <BreadcrumbItem>
+                <Input
+                  id="fileName"
+                  value={fileName}
+                  size={20}
+                  onInput={(event: ChangeEvent<HTMLInputElement>) => {
+                    setFileName(event.currentTarget.value)
+                  }}
+                  ref={_ref => (inputRef.current = _ref)}
+                  onBlur={rebuildPaths}
+                  onFocus={({ target }) => {
+                    const value = (parentPath ? parentPath + FILE_SEPERATOR : '') + fileName
+                    setFileName(value)
+                    setParentPath('')
+                    setTimeout(() => {
+                      target.setSelectionRange(value.length, value.length)
+                      target.scrollLeft = Number.MAX_SAFE_INTEGER
+                    }, 0)
+                  }}
+                  placeholder="Name your file..."
+                />
+              </BreadcrumbItem>
+            </ButtonGroup.Root>
+          </ListActions.Left>
+          <ListActions.Right>
+            <Button variant="default" size="sm" onClick={() => setIsCommitDialogOpen(true)}>
+              Commit Changes
+            </Button>
+            <Button
+              variant="outline"
+              size="sm"
+              onClick={() => {
+                handleCancelFileEdit()
               }}
-            />
-          ) : (
-            <CodeDiffEditor
-              language={language}
-              original={originalFileContent}
-              modified={content}
-              themeConfig={themeConfig}
-              options={{
-                readOnly: true
+            >
+              Cancel
+            </Button>
+          </ListActions.Right>
+        </ListActions.Root>
+        <Spacer size={5} />
+        <StackedList.Root onlyTopRounded borderBackground>
+          <StackedList.Item disableHover isHeader className="px-3 py-2.5">
+            <ToggleGroup
+              onValueChange={(value: ViewTypeValue) => {
+                if (value) {
+                  setView(value)
+                }
```

Comment:
Can this be something falsey? Can you just call it?
```suggestion
              onValueChange={(value: ViewTypeValue) => setView(value)}
```

or even
```suggestion
              onValueChange={setView}
```